homeassistant/components/homeassistant_alerts/__init__.py

```diff
@@ -100,7 +102,19 @@ def async_schedule_update_alerts() -> None:
 
     coordinator = AlertUpdateCoordinator(hass)
     coordinator.async_add_listener(async_schedule_update_alerts)
-    await coordinator.async_refresh()
+
+    refresh_debouncer = Debouncer(
```

Comment:
Why do we need this? Isn’t it fine to just pick up new integrations at next refresh?

Reply:
OK, we could do that, but it's a different PR at that point.
The downside is that we have to wait until after the next refresh for alerts for a newly added config entry.